src/ngraph/frontend/onnx_import/op/lp_pool.cpp

```diff
@@ -39,7 +39,16 @@ namespace ngraph
                 {
                     const std::shared_ptr<ngraph::Node> data{node.get_ng_inputs().at(0)};
                     const std::size_t channel_axis{1};
-                    const std::size_t channels_count = data->get_shape().at(channel_axis);
+
+                    const auto data_shape = data->get_output_partial_shape(0);
+                    NGRAPH_CHECK(data_shape.rank().is_static(),
+                                 "Rank of input data must be static");
+                    NGRAPH_CHECK(data_shape[0].is_static(),
+                                 "First dimension of input data must be static");
+                    NGRAPH_CHECK(data_shape[channel_axis].is_static(),
```

Comment:
Not sure but for safety, should we also check that rank is at least(or exactly equal to) 2?

Reply:
done